### PR TITLE
[corda] update readme.md files

### PR DESCRIPTION
**Pull request by nikhilagolla**
Signed-off-by: nikhilagolla <nikhila.golla@accenture.com>

**Changelog**
- Updated README.md files of charts and roles.

 

**Reviewed by**
@suvajit-sarkar @jagpreetsinghsasan  @sownak 

 

**Linked issue**
#1617

### CODE REVIEW

File: platforms/r3-corda/releases/dev/carrierniki-ns/namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```
Comment (sownak) [inline platforms/r3-corda/releases/dev/carrierniki-ns/namespace.yaml:1]: Please do not check-in any release files

**Review by sownak — CHANGES_REQUESTED** [review]

File: platforms/shared/configuration/site.yaml [hunk 2]
```diff
@@ -5,8 +5,8 @@
   - hosts: all
     gather_facts: no
     no_log: "{{ no_ansible_log | default(false) }}"
-  - import_playbook: network-schema-validator.yaml  
-    when: reset is undefined or reset == 'false'       
+  #-import_playbook: network-schema-validator.yaml
```
Comment (jagpreetsinghsasan) [inline platforms/shared/configuration/site.yaml:8]: Please revert this change.

**Review by sownak — APPROVED** [review]
